include/sys/spa.h

```diff
@@ -971,6 +971,8 @@ extern int spa_import_progress_set_max_txg(uint64_t pool_guid,
     uint64_t max_txg);
 extern int spa_import_progress_set_state(uint64_t pool_guid,
     spa_load_state_t spa_load_state);
+extern void spa_import_progress_set_notes(uint64_t pool_guid,
+    const char *notes);
```

Comment:
I see most of spa_import_progress_*() functions take pool_guid argument.  I guessed it supposed to cover some activity before spa created, but as I can see all of them use spa_guid(spa) to obtain it.  Are there such ones potentially or we could simplify the code?  IIRC user-space may take time to scan disks for labels before passing config to kernel. At that point we likely don't have spa, but I am not sure how this project covers that area.

Reply:
Changed to take a `spa_t` argument instead of guid